test/unit/vector_tile.test.cpp

```diff
@@ -112,3 +112,19 @@ TEST_CASE( "Prevent massive over allocation" ) {
     mapbox::vector_tile::points_arrays_type geom = feature.getGeometries<mapbox::vector_tile::points_arrays_type>(1.0);
     REQUIRE(geom.capacity() <= 655360);
 }
+
+TEST_CASE(" Prevent underflow in case of missing commands" ) {
+    std::string buffer = open_tile("test/test2048.mvt");
+    mapbox::vector_tile::buffer tile(buffer);
+    auto const layer_names = tile.layerNames();
+    REQUIRE(layer_names.size() == 1);
+    REQUIRE(layer_names[0] == "roads");
+    auto const layer = tile.getLayer("roads");
+    REQUIRE(layer.featureCount() == 243);
+    for (std::size_t i = 0; i < layer.featureCount(); ++i) {
+        auto const feature = mapbox::vector_tile::feature(layer.getFeature(i), layer);
+        REQUIRE(feature.getType() == mapbox::vector_tile::GeomType::LINESTRING);
+        mapbox::vector_tile::points_arrays_type geom = feature.getGeometries<mapbox::vector_tile::points_arrays_type>(1.0);
+        REQUIRE(!geom.empty());
```

Comment:
can combine these or at least `auto` the getGeometries return

Reply:
👍 https://github.com/mapbox/vector-tile/pull/61/commits/fcb9479e5d8b6be67bf2fe99de3fab9107c321a9